website/app/templates/head.hbs

```diff
@@ -45,6 +45,7 @@ We've made some small customizations to simplify this compared to the default te
 {{! BEGIN TWITTER }}
 
 <meta name="twitter:card" content="summary" />
+<meta name="twitter:site" content="@HashiCorp" />
```

Comment:
```suggestion

{{#if this.model.twitterUsername}}
  <meta name="twitter:site" content={{this.model.twitterUsername}}>
  <meta name="twitter:creator" content={{this.model.twitterUsername}}>
{{/if}}
```

For consistency in following the way `ember-meta` [prescribes things](https://github.com/shipshapecode/ember-meta#global-config) can you add this value as a property in `config/environment.js` with the other `ember-meta` config and then add back in this logic which I had previously removed.

Reply:
Made this update along with reordering